src/tribler-core/tribler_core/modules/ipv8_module_catalog.py

```diff
@@ -104,7 +104,7 @@ class PopularityCommunityLauncher(IPv8CommunityLauncher):
 @overlay('tribler_core.modules.metadata_store.community.gigachannel_community', 'GigaChannelCommunity')
 @kwargs(metadata_store='session.mds', notifier='session.notifier')
 @walk_strategy('ipv8.peerdiscovery.discovery', 'RandomWalk')
-@walk_strategy('tribler_core.modules.metadata_store.community.sync_strategy', 'SyncChannels')
+@walk_strategy('tribler_core.modules.metadata_store.community.sync_strategy', 'SyncChannels', target_peers=-1)
```

Comment:
It would be nice to have a corresponding const for `-1` to clarify what exactly is meant by `-1`

Reply:
I have two variations for you:

a) A `-1` constant:
```python
NEVER_ENOUGH = -1
 ...
@walk_strategy('tribler_core.modules.metadata_store.community.sync_strategy', 'SyncChannels',
               target_peers=NEVER_ENOUGH)
```
b) A `dict` mapping:

```python
NEVER_STOP = {'target_peers': -1}
 ...
@walk_strategy('tribler_core.modules.metadata_store.community.sync_strategy', 'SyncChannels',
               **NEVER_STOP)
```

I am also open to other naming of the variable - please let me know what you prefer. At any rate, I'm not the right person to pick the variable name: I'm way too accustomed to `-1` and the inner workings of IPv8.

Reply:
Thank you, @qstokkink! Both variants look very attractive.
I'm personally like the first one more than the second one.

About the name, maybe something from:
```python
INFINITE = -1
ENDLESS = -1 
```

Reply:
`INFINITE = -1` would be the most natural option then to me.

Reply:
One moment, I'll make a PR.